<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainmahasiswa</class>
 <widget class="QMainWindow" name="Mainmahasiswa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>281</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="pageTambah">
     <widget class="QPlainTextEdit" name="outputdaftarmatkul">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addmatkul">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTambahMatkul">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>tambah matkul</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="writenim">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Index</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageTampil">
     <widget class="QPlainTextEdit" name="outputtampilmatkul">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>191</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTampilMatkul_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Tampil matkul</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="showmatkul">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageUndo">
     <widget class="QPushButton" name="btnUndoMatkul">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo Matkul</string>
      </property>
     </widget>
     <widget class="QLabel" name="keteranganundo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="undoperson">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btnTotalSks_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Sks</string>
      </property>
     </widget>
     <widget class="QLabel" name="outputtotal">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NIM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>59</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnPageTambahMatkul">
       <property name="text">
        <string>tambah matkul</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPageUndoMatkul">
       <property name="text">
        <string>undo matkul</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTampilMatkul">
       <property name="text">
        <string>tampilkan matkul</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTotalSks">
       <property name="text">
        <string>Total sks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKembali">
       <property name="text">
        <string>kembali</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
